### ISSUE TITLE
Missing File: UniformCalculator.h

### ISSUE THREAD

**Issue by IAmTheEye** [adamgit/GL2KitExtensions#1]
Hi Adam,

I like your Open GL tutorial series - used i t a while ago, and it's a very nice refresh. Your current code in GitHub misses the UniformCalculator.h file - could you add it?

**Comment by adamgit** [comment]
You don't need it. It's for a future article.

**Comment by IAmTheEye** [comment]
Ok, it just doesn't compile, I didn't check whether it's used yet - and indeed it works when I comment it out. Thanks!

**Comment by adamgit** [comment]
OK, ahort answer: only use the files that are linked in the articles so far.

Longer answer: I'm
Debugginf the code for the next article, so some of these classes are
probablu half complete sorry. I will have anoter look soon

**Comment by adamgit** [comment]
Yep - delete lines 7 and 45 of GLK2DrawCall.m that reference that file -
they aren't used anywhere else in the project right now. In the next
article, I'm looking to delete / replace that class with something better.
